### PR TITLE
Add section on obsoleted stats.

### LINKED ISSUE #174: Need advice for handling obsolete stats
If stats are renamed or declared useless, we need to keep the names around for reference.
There should be advice in the spec on how to do this.

### PR DESCRIPTION
**Pull request by alvestrand**
Closes #174

### CODE REVIEW

File: webrtc-stats.html [hunk 1]
```diff
@@ -2096,6 +2117,36 @@
     </section>
     <section>
       <h2>
+        Obsolete stats
+      </h2>
+      <p>
+        This section's content is an example.
```
Comment (vr000m) [inline webrtc-stats.html]: We have a concrete example, rtt got deprecated for roundTripTime, handle that as a separate issue?

**Comment by alvestrand** [comment]
Harald will switch example in the "obsoleted" section to use rtt (which was renamed and moved).
Moved in #167 - renamed in #95 
We should say that we don't keep track of stuff that was never implemented - if we were pedantic, we would report things that were in drafts, but we don't want to do that.

**Comment by alvestrand** [comment]
@vr000m still LGTM?

**Comment by vr000m** [comment]
The #167 is related to the RTP RTT, we need another entry for RTCInboundRTPStreamStats.

In the case of #167, we changed the units from ms to seconds.

**Pull request merged**